buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -770,6 +770,10 @@ class BeamModulePlugin implements Plugin<Project> {
         + (defaultLintSuppressions + configuration.disableLintWarnings).collect { "-Xlint:-${it}" })
       }
 
+      project.tasks.withType(Jar).configureEach {
+        preserveFileTimestamps(false)
+      }
+
```

Comment:
Ah yes I guess applyJavaNature is called in applyPortabilityNature already so this caputres both :+1: